src/Uno.UI/UI/Xaml/Controls/Popup/PopupRoot.cs

```diff
@@ -116,7 +106,6 @@ protected override Size ArrangeOverride(Size finalSize)
 			ArrangeElement(child, new Rect(new Point(), finalSize));
 		}
 
-		DispatcherQueue.TryEnqueue(() => Arranged?.Invoke());
```

Comment:
Curious why?

Reply:
Why what? I originally added this event so that when popups were rearranged, we would update the clip of the native elements. Now that the implementation reads directly from the visual of the popup during the render cycle, it will always get updated automatically on every render.